recipes/micromamba/LIBSOLV_LICENSE.txt

```diff
@@ -0,0 +1,26 @@
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions
+are met:
+
+1. Redistributions of source code must retain the above copyright
```

Comment:
This file doesn't actually contain the copyright

Reply:
I copied the LICENSE file straight from the repo. I added the copyright notice now